homeassistant/components/aemet/const.py

```diff
@@ -206,7 +206,7 @@
         name="Condition",
     ),
     SensorEntityDescription(
-        key=ATTR_FORECAST_PRECIPITATION,
```

Comment:
Changing these keys, will change the unique_id of the `sensors` provided by this integration (and thus a breaking change).

The breaking change should be avoided. Either by not renaming the keys, or by providing a migration for existing entity entries in the registry.